x-pack/legacy/plugins/apm/public/components/shared/DatePicker/index.tsx

```diff
@@ -14,6 +14,16 @@ import { useUrlParams } from '../../../hooks/useUrlParams';
 export function DatePicker() {
   const location = useLocation();
   const { urlParams, refreshTimeRange } = useUrlParams();
+  const commonlyUsedRanges = [
+    { start: 'now-15m', end: 'now', label: 'Last 15 minutes' },
+    { start: 'now-30m', end: 'now', label: 'Last 30 minutes' },
+    { start: 'now-1h', end: 'now', label: 'Last 1 hour' },
+    { start: 'now-24h', end: 'now', label: 'Last 24 hours' },
+    { start: 'now-7d', end: 'now', label: 'Last 7 days' },
+    { start: 'now-30d', end: 'now', label: 'Last 30 days' },
+    { start: 'now-90d', end: 'now', label: 'Last 90 days' },
+    { start: 'now-1y', end: 'now', label: 'Last 1 year' }
```

Comment:
I should have added to the issue description: all user-facing strings should be wrapped in `i18n.translate` to enable translation.

Example:
```js
import { i18n } from '@kbn/i18n';

 const commonlyUsedRanges = [{ 
  start: 'now-1y', 
  end: 'now', 
  label: {i18n.translate('xpack.apm.datePicker.last1Year', {
    defaultMessage: 'Last 1 year'
  })} 
}];
```

Reply:
I see most aliases have `Label` postfix.
Should I put something like 
```javascript
i18n.translate('xpack.apm.datePicker.last15MinutesLabel' ...
i18n.translate('xpack.apm.datePicker.last1HourLabel' ...
```
?

Reply:
Oh yeah. The label suffix is a good addition 👍